targets/nvidia-jetson-orin.nix

```diff
@@ -51,13 +42,17 @@
           ../modules/virtualization/microvm/microvm-host.nix
           ../modules/virtualization/microvm/netvm.nix
           {
+            users.users."ghaf".extraGroups = ["docker"];
             ghaf = {
               hardware.nvidia.orin.enable = true;
               hardware.nvidia.orin.somType = som;
-
+              #virtualization.docker.daemon.enable = true;
```

Comment:
commenting out is probably not intentional?

Reply:
It will be removed.